cdap-watchdog/src/main/java/co/cask/cdap/metrics/process/MigrationTableHelper.java

```diff
@@ -31,10 +31,16 @@
  */
 final class MigrationTableHelper {
 
+  private static volatile boolean stopping = false;
```

Comment:
@anew Am not too sure about this approach, will appreciate your suggestion if this is okay